ftl/cached/ftl_python_cached_yaml.py

```diff
@@ -0,0 +1,72 @@
+"""A script to generate a cloudbuild yaml."""
+
+import os
+import yaml
+
+_TEST_DIR = '/workspace/ftl/python/testdata'
```

Comment:
Could you refactor the three new cache .py files into one script that takes the language as a parameter?

Reply:
done